platforms/substrate/configuration/roles/create/genesis/tasks/nested_main_genesis.yaml

```diff
@@ -0,0 +1,35 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Generate genesis helmrelease file
+- name: "Create genesis helmrelease file"
+  include_tasks: nested_genesis.yaml
+  loop: "{{ peers }}"
+  loop_control:
+    loop_var: peer
+  when: peer.type == 'bootnode'
```

Comment:
Ok, seems you have done this, but the loops are all wrong, this will create too many unnecessary loops